packages/agent/src/dicom.test.ts

```diff
@@ -13,7 +13,7 @@ let endpoint: Endpoint;
 describe('DICOM', () => {
   beforeAll(async () => {
     console.log = jest.fn();
-    dimse.log.transports.forEach((t) => (t.silent = true));
+    dimse.log.disableAll(false);
```

Comment:
Nice 👍 

I saw the build error and got scared that it was going to be much more complicated than this